typescript/commands/build.py

```diff
@@ -23,7 +25,7 @@ def run(self):
             else:
                 sublime.active_window().show_input_panel(
                     "Build parameters: ",
-                    "",  # initial text
+                    self.build_parameters,
```

Comment:
isn't ``build_parameters`` a class property instead of an instance property? I thought it can only be referenced by ``TypescriptBuildCommand.build_parameters``